pkg/pillar/containerd/containerd.go

```diff
@@ -90,6 +92,11 @@ func InitContainerdClient() error {
 	}
 	if contentStore == nil {
 		contentStore = CtrdClient.ContentStore()
+		// see if we need to fine-tune default snapshotter based on what flavor of storage persist partition is
+		persistType, err := ioutil.ReadFile(eveStorageTypeFile)
+		if err == nil && string(persistType) == "zfs" {
```

Comment:
If we expect folks to manually edit this file you should probably strip whitespace like the CR at the end to make the comparison work.

Reply:
Great catch @eriknordmark -- will fix this